app.py

```diff
@@ -37,7 +37,24 @@ def create_story():
             "families": {
                 story["familyId"]: True
             }
-        })
+        })        
         response = app.response_class(
             response=json.dumps(new_story.get()), status=201, mimetype="application/json")
         return response
+
+
+@app.route('/users', methods=['POST'])
+@cross_origin()
+def create_user():
+    if request.method == "POST":
+        request_data = request.get_json()
+        users = db.reference("users/" + request_data["userId"])
+        res = users.set({
+           	"email": request_data["email"], 
+	        "fullName": request_data["fullName"],
+	        "displayName": request_data["displayName"],
+	        "familyName": request_data["familyName"],
+            "fid": str(uuid.uuid4()),
```

Comment:
This doesn't look like it matches the schema for a user, it should be

```
families: { the-unique-fam-id: true }
```

The unique family id should be generated outside the object and used to create a new family (along with the family name). We should not be storing the fam name on users object.

Remember, this function is being called on registration, currently this is the first member of a family signing up.